README.md

```diff
@@ -1 +1,119 @@
-# sample_project_with_webpack
\ No newline at end of file
+# Todo List app built with webpack
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [About the Project] <a name="about-project"></a>
+
+**[awesome books]** is a project that displays a list of books.
```

Comment:
- [OPTIONAL] I would recommend you to always make your about project to be inline with the title of your project. This milestone is `Todo-List`, project not `awesome books` project.